### PR TITLE
feat: make templates consistent across runtimes

### PR DESCRIPTION
**Pull request by lance**
# Changes

- 🎁  Ensures that the templates for all runtimes are consistent with #572 
  - Highlight where to put business logic
  - Effective, useful and descriptive comments
  - Template functions echo their input
  - All templates have unit tests
-

/kind enhancement

Fixes: https://github.com/knative-sandbox/kn-plugin-func/issues/572

**Release Note**

```release-note
Aligns all templates to a common standard behavior and comments
```

### CODE REVIEW

File: templates/python/cloudevents/func.py [hunk 1]
```diff
@@ -8,8 +8,8 @@ def main(context: Context):
     The context parameter contains the Flask request object and any
     CloudEvent received with the request.
     """
-    # print(f"Method: {context.request.method}")
+    # Add your business logic here
```
Comment (zroubalik) [inline templates/python/cloudevents/func.py:11]:
```suggestion

    # Add your business logic here
```

File: templates/quarkus/cloudevents/src/main/java/functions/Function.java [hunk 2]
```diff
@@ -4,10 +4,20 @@
...
 
+    /**
+     * Use the Quarkus Funq extension for the function. This example
+     * function simply echoes its input data.
+     * @param input a CloudEvent
+     * @return a CloudEvent
+     */
     @Funq
     public CloudEvent<Output> function(CloudEvent<Input> input) {
+        // Add your business logic here
```
Comment (zroubalik) [inline templates/quarkus/cloudevents/src/main/java/functions/Function.java:20]:
```suggestion

        // Add your business logic here

```

File: templates/quarkus/http/src/main/java/functions/Function.java [hunk 3]
```diff
@@ -2,10 +2,19 @@
...
 public class Function {
 
+    /**
+     * Use the Quarkus Funqy extension for our function. This function simply echoes its input
+     * @param input a Java bean
+     * @return a Java bean
+     */
     @Funq
     public Output function(Input input) {
+        // Add business logic here
```
Comment (zroubalik) [inline templates/quarkus/http/src/main/java/functions/Function.java:17]:
```suggestion

        // Add business logic here

```

**Review by zroubalik — COMMENTED** [review]
Looking good, just a small nit. I'd add blank lines around the comment, to make it more prominent and easily recognizable for users when the open the source code for the first time.

**Comment by matejvasek** [comment]
@lance there are style issues, and the embedded templates contains some `target/*` files that shouldn't be there.

**Comment by matejvasek** [comment]
There is trailing white space in `templates/python/cloudevents/func.py:11:`.

**Pull request merged**